roles/confluent.ssl/tasks/create_keystore_and_truststore.yml

```diff
@@ -4,15 +4,16 @@
     path: /var/ssl/private/generation
     state: absent
 
-- name: Delete Old Keystore
+- name: Delete Old Certificate Files
   file:
-    path: "{{keystore_path}}"
-    state: absent
-
-- name: Delete Old Truststore
-  file:
-    path: "{{truststore_path}}"
+    path: "{{item}}"
     state: absent
+  loop:
+    - "{{keystore_path}}"
```

Comment:
Would it make sense to limit this only to files we are generating? We save a bit on not copying stuff that is copied in the next step and may be the same.

Reply:
The code now generates keystore, truststore, cert, key and ca cert for each host
I suppose the filename could be changed to create_certs.yml to be clearer. I can change it if that helps

I've found w openssl and keytool commands its much easier to start fresh. 
You are right in the case of provided keystores you would not need to delete before copy because the copy should overwrite. 

In the other cases its a good idea to delete the keystore before regenerating them. You usually get file already exists errors.